### PR TITLE
spaces after # in code comments, text edits to TS01

### PR DESCRIPTION
**Pull request by mjones01**

### CODE REVIEW

File: TS01-Time-Series-Metadata.Rmd [hunk 1]
```diff
@@ -235,83 +246,87 @@ the title of a dataset might be embedded in a `<title>` tag as follows:
...
 
 The `EML` package for `R` is designed to read and allow users to work with `EML`
-formatted metadata. In this tutorial, we will overview demonstrate how we can
-use EML in an automated workflow. NOTE: To save time, we will not explicetedly 
-teach the `EML` package given it is still being developed. But we will provide
-an example of how you can access `EML` programmatically below.
+formatted metadata. In this tutorial, we demonstrate how we can use EML in an
+automated workflow. 
+
+NOTE: The `EML` package is still being developed, therefore we will not
```
Comment (lwasser) [inline TS01-Time-Series-Metadata.Rmd:257]: This is a nice addition - thank you @mjones01 !

File: TS03-Subset-Data-and-No-Data-Values-In-R.Rmd [hunk 2]
```diff
@@ -386,48 +382,45 @@ and y-axes. Also give the plot a title!
...
 
-
-#4. Check for NoData values
+# 4. Check for NoData values
 sum(is.na(harMet.daily$date))
 sum(is.na(harMet.daily$airt))
 sum(is.na(harMet.daily$prec))
 sum(is.na(harMet.daily$part))
-#OuputNote: PART is missing 1032 values
+# Ouput Note: PART is missing 1032 values
```
Comment (lwasser) [inline TS03-Subset-Data-and-No-Data-Values-In-R.Rmd]: @mjones01 should this be "output" ?

File: TS05-Plotting-Time-Series-ggplot-In-R.Rmd [hunk 3]
```diff
@@ -637,7 +639,7 @@ grid.arrange(AirTempDaily, AirTempMonthly, ncol=2)
 In this lesson, we've covered the basics of `ggplot`. There are many great 
 resources the cover refining `ggplot` figures. A few are below:
 
-1. ggplot2 Cheatsheet from Zev Ross: <a href="http://neondataskills.org/cheatsheets/R-GGPLOT2/" target="_blank"> ggplot2 Cheatsheet</a>  
+1. ggplot2 Cheatsheet from Zev Ross: <a href="http://zevross.com/blog/2014/08/04/beautiful-plotting-in-r-a-ggplot2-cheatsheet-3/" target="_blank"> ggplot2 Cheatsheet</a>
```
Comment (lwasser) [inline TS05-Plotting-Time-Series-ggplot-In-R.Rmd]: thank you for catching this.

**Comment by lwasser** [comment]
These edits look great, @mjones01 . please change the very minor typos that I found. Then, if you feel comfortable with it, rebuild the repo and push the full build to this repo.

**Comment by mjones01** [comment]
I'll change/ re-review these after the vector series. 

Question: Most other live pages use lowercase for permalinks.  This series uses capitals, it is not yet live but I did just find a link to the metadata lesson here which has a new permalink & title. Since I need to go through Raster & Vector and look to see where it is linked anyway, should we change this series to all lowercase permalinks?

**Comment by lwasser** [comment]
Yes, let's use lowercase throughout. it's easier to remember to keep urls lower case!
Thank you

**Comment by mjones01** [comment]
Do we make this change on pages that have gone live?  (At least Raster 00 and 01)

**Comment by lwasser** [comment]
oh shoot -- good call. how many lessons in raster have caps? it will impact our analytics. it could impact comments too. My suggestion is NO for now. Only time series can be fixed as we aren't live.

**Comment by mjones01** [comment]
Which ones: 
Raster: all except 02 -- AND none have comments enabled!
Vector: none (with exception of R capitalized)
rest of site: didn't check all pages but what I looked at are a mix - DO NOT CHANGE (given history)

gAnalytics: we can set a filter so that all pages with same text regardless of capitalization are displayed together.  

Given the fact that comments AREN'T enabled and we can make a fix for gAnalytics I think we SHOULD change them in Raster on this next review (we'll have to push up the comments change anyway).  

Only question now (raster & going forward), is even with all lowercase, is R also lowercase? (Comments are enabled for Vector, so I'm not suggesting with change them even if we go with a lowercase R).

**Comment by lwasser** [comment]
Check the live site. Comments are enabled. We just turned them off locally before I did it globally in the config file.
Let's leave it . Trust me, we don't' want to get into GA filters for these things. They are a pain to manage.

From here on in let's stay lower case. This is a minor change so let's not spend a lot of time on it given it will be more effort than it's worth. Thanks Megan.

**Comment by mjones01** [comment]
Looking at the live site I still see no comment capabilities at the bottom of any raster page; but I do on vector pages.

**Comment by lwasser** [comment]
Weird. I see that in the code too. Ok I know why I am confused. I received a comment the other day on raster lessons Megan - please I don't want to change URL's to pages that are live. it's not worth this much time to discuss it. It's bad practice to change a URL once it's published for many reasons. Please stop pushing this issue. People are hitting the pages and we have caps throughout the rest of data skills.

**Comment by lwasser** [comment]
is this ready to merge? If so, i'll try to do it Monday. Signing off in a few.

**Comment by mjones01** [comment]
@lwasser I have reviewed and pushed up my .Rmd files.  I'm concerned about the inclusion of the EML package.  The code is not working and it appears to be due to the package still being in development. 
- eml_read() is changed to read_eml() : I can't find this change documented anywhere but based on our current github.io build eml_read() did work.  It doesn't now, the EML package can't find that function & the 'readme' in the EML package github only mentions read_eml().  I got read_eml() to load the file but it too nearly 75 seconds to load (more than "a few").  
- eml_get() isn't working and I didn't find a solution 

If the package is still at this stage of development should we cut that second half of the lesson for now?

**Comment by mjones01** [comment]
I was going to accept the merge so that you could work on these updated .Rmds while you check out the TS01 issues.  However, since you said you'd do it on Monday I'll let you.

File: TS01-Time-Series-Metadata.Rmd [hunk 4]
```diff
@@ -337,47 +349,44 @@ eml_get(eml_HARV,"coverage")
 
 ##Identify & Map Data Location
```
Comment (lwasser) [inline TS01-Time-Series-Metadata.Rmd:349]: @mjones01 missing a space for a header? ## text vs ##text

File: TSCulmination-Work-With-NDVI-and-Met-Data-In-R.Rmd [hunk 5]
```diff
@@ -26,15 +26,15 @@ comments: true
```
Comment (lwasser) [inline TSCulmination-Work-With-NDVI-and-Met-Data-In-R.Rmd:26]: let's call this 07-Culmination-Work-With-NDVI-and-Met-Data-In-R.Rmd

**Comment by mjones01** [comment]
Okay.  I've made the changes above and rebuilt the full site so the name changes carried throughout).  

The EML portion of 01 still needs to be reviewed and revised/removed by @lwasser .  Then the repo needs to be rebuilt for a final review (in Dev?).

**Comment by lwasser** [comment]
Ok - I will get to it. I'm going to work on the intro lessons for a bit this afternoon.  EML is covered there as well so I can address the issues and then apply that to the EML lesson in time series. This is on my list to do!

Leah

**Comment by mjones01** [comment]
Updated URLs/permalinks as discussed for Raster (final / ).

**Pull request merged**